release-notes.txt

```diff
@@ -0,0 +1,224 @@
+BISDN Linux
+v4.5.0 Release Notes
+Date: 2022-03-14
+
+! New in this release
+- OF-DPA: We now support creating and enabling KNET interfaces.
+- baseboxd:
+  - updated version to 1.11.1
+  - fixed several minor bugs related to bonds and terminaton MACs.  - some STP issues were fixed
+- Linux kernel updated to version 5.15.22
+- mstpd: move to a patched fork of of the upstream project.
```

Comment:
This has (or should not) have any functional impact, so I don't think this needs to be in the highlights part.

Reply:
removed mstp part